index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <a class="name">Muh Noman</a>
+      <img class="navi" src="/images/navi.png" alt="navigation" />
+    </header>
+
+    <main>
+      <section>
+        <h1>Hey There. I’m Noman</h1>
+        <h6>I’am a Software Developer</h6>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+
+        <ul>
+          <li>
+            <a href="#"><img src="/images/github.png" alt="github" /></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="/images/mvector.png" alt="In sign" /></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="/images/victory.png" alt="victory" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/twitter.png" alt="twitter" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="/images/mvector.png" alt="M sign" /></a>
+          </li>
+        </ul>
+
+        <img
+          class="illustraion"
+          src="/images/Group.png"
+          alt="illustration design"
+        />
```

Comment:
- [ ] [OPTIONAL] please make sure that this background image is created using CSS `background-image` and not by using an `img` tag.

Reply:
please give me relaxation at this time because I do not have too much time. I will try next time. thanks